### PR TITLE
OHRI 1419: Renamed Programs to Programmes

### PR DESCRIPTION
**Pull request by arodidev**
Renaming of programs to Programmes in the sidebar
Screenshot 2023-06-13 at 19 22 18

### CODE REVIEW

**Comment by ODORA0** [comment]
@arodidev Add screenshots of the change too

**Comment by arodidev** [comment]
> 

@ODORA0 Screenshot added

**Review by pirupius — CHANGES_REQUESTED** [review]

File: packages/esm-ohri-core-app/src/ohri-dashboard/side-menu/ohri-dashboard-side-nav.component.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ const OHRIDashboardSideNav = () => {
       <SideNavItems>
         <ExtensionSlot name="dashboard-links-slot" />
 
-        <p className={styles.sideNavTextHeader}>{t('programs', 'Programs')}</p>
+        <p className={styles.sideNavTextHeader}>{t('programmes', 'Programmes')}</p>
```
[the remark this answers is not in the document]
Comment (arodidev) [reply packages/esm-ohri-core-app/src/ohri-dashboard/side-menu/ohri-dashboard-side-nav.component.tsx:14]: @pirupius Well understood. So would this approach be better?

`<p className={styles.sideNavTextHeader}>{t('programs', 'Programmes')}</p>`

**Comment by pirupius** [comment]
@arodidev why did u close this PR?

**Pull request closed without merging**